vault/logical_system_activity.go

```diff
@@ -21,6 +21,14 @@ import (
 // defaultToRetentionMonthsMaxWarning is a warning message for setting the max retention_months value when retention_months value is more than activityLogMaximumRetentionMonths
 var defaultToRetentionMonthsMaxWarning = fmt.Sprintf("retention_months cannot be greater than %d; capped to %d.", activityLogMaximumRetentionMonths, activityLogMaximumRetentionMonths)
 
+const (
+	// WarningCurrentBillingPeriodDeprecated is a warning string that is used to indicate that the current_billing_period field, as the default start time will automatically be the billing period start date
+	WarningCurrentBillingPeriodDeprecated = "current_billing_period is deprecated; unless otherwise specified, all requests will default to the current billing period"
```

Comment:
Strictly speaking this wasn't needed but it seemed to make sense to include since there was no harm and future backports might be easier as a result.

Reply:
Actually since the change involving deprecation of the current_billing_period wasn't backported + we don't have any intention of backporting because we're gonna force them to upgrade for this to happen

- let's remove this so there's no confusion in the future.

Reply:
Sounds good! Will do.

Reply:
Done!